lib/publiccloud/azure.pm

```diff
@@ -141,6 +162,7 @@ sub upload_img {
     $img_name =~ s/\.vhdfixed/-$sku.vhd/;
     my $disk_name = $img_name;
     my $storage_account = get_var('PUBLIC_CLOUD_STORAGE_ACCOUNT', 'eisleqaopenqa');
+    my $container = $self->container;
```

Comment:
why a dedicated variable is needed ? Why not to directly use `$self->container` ?